jsettlers.graphics/src/jsettlers/graphics/image/ImageIndexImage.java

```diff
@@ -31,6 +38,30 @@
 	private final float umax;
 	private final float vmax;
 
+	private static final float[] TEMP_BUFFER = new float[5 * 6];
```

Comment:
@michaelzangl: Didn't we agree on naming such "not really constant" variables in normal camel case?